pep-0001.txt

```diff
@@ -281,14 +281,25 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The term "PEP-Delegate" is used under the Steering
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "PEP-Delegate" field in the PEP's header. For
+PEPs written prior to the Steering Council's governance model, the field name
```

Comment:
```suggestion
PEPs written prior to the Steering Council governance model, the field name
```

Nitpick, but the possessive implies that the Steering Council owns/posses the governance model, as opposed to itself being the name of the governance model